### PR TITLE
chore(warnings): Mute warnings during build

### PR DESCRIPTION
**Pull request by trollepierre**
While building the app, a warning is displayed mentioning webpack cannot require conditionnally. 
This MR fixed that, adds a test on this fix and keep preventing error attached below.

### CODE REVIEW

File: src/drive/web/modules/services/components/iconsContext.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const iconsContext = require.context(
```
Comment (trollepierre) [inline src/drive/web/modules/services/components/iconsContext.js:1]: oups:
Capture d’écran 2022-01-11 à 17 50 39

Comment (Crash--) [reply src/drive/web/modules/services/components/iconsContext.js:1]: I don't understand if I still need to read this PR or not with this screenshot?

Comment (trollepierre) [reply src/drive/web/modules/services/components/iconsContext.js:1]: The PR was in Draft when you saw it. Now the error above is fixed in the PR and the PR is ready for review.

**Comment by trollepierre** [comment]
close for https://github.com/cozy/cozy-drive/pull/2476

**Pull request closed without merging**